### PR TITLE
M0078DDG-493 NLS fallback locale + default locale

### PR DESCRIPTION
**Pull request by fpinel-microej**

### CODE REVIEW

File: ApplicationDeveloperGuide/nls.rst [hunk 1]
```diff
@@ -99,6 +99,48 @@ messages, for example:
 
    String label = Labels.NLS.getMessage(Labels.Label1);
 
+For the application to know which language to use among those made available and when, you can set it and change it at any point using the `setCurrentLocale(locale)`_ method. 
+If no locale has been set yet when getting a message, the translation for the first locale available in alphabetical order will be used by default. 
+However, you can also pick this locale to default to yourself, by adding a `com.microej.binarynls.defaultLocale` property followed by a locale name in a ``.properties.list`` file. 
+
+
+.. _setCurrentLocale(locale): https://repository.microej.com/javadoc/microej_5.x/apis/ej/nls/NLS.html#getCurrentLocale--
```
Comment (seonmej) [inline ApplicationDeveloperGuide/nls.rst:107]: Beware, it references the getter instead of the setter.

Comment (fpinel-microej) [reply ApplicationDeveloperGuide/nls.rst]: Fixed, thanks. Must have been a clipboard leftover from the `NLS.getCurretLocale` link below.

**Review by seonmej — APPROVED** [review]

File: ApplicationDeveloperGuide/nls.rst [hunk 2]
```diff
@@ -99,6 +99,48 @@ messages, for example:
...
+
+Dealing With Missing Translations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, if a translation is missing for a given ``msgid`` in a PO file in a given language, the message returned by the `ej.nls.NLS.getMessage()`_ method with the locale set to this language will simply be the ``msgid`` itself. 
+In the case of an XML Android String resource, the ``name`` attribute of a missing ``string`` element will be returned. 
+However if returning this identifier is not a suitable solution, you might want to set a fallback locale parameter for an interface. 
+This parameter corresponds to a language to print the translation for a message in, in case it is not available in the current language.
+
+You can set this fallback locale by specifying a locale name in a ``.nls.list`` file, after the name of the interface you want this locale to be the fallback for, separated by a colon ``:``.
```
Comment (frivieremicroej) [inline ApplicationDeveloperGuide/nls.rst:118]: Indicate the version
```suggestion
Starting with version XXX, you can set this fallback locale by specifying a locale name in a ``.nls.list`` file, after the name of the interface you want this locale to be the fallback for, separated by a colon ``:``. 
```

**Review by frivieremicroej — CHANGES_REQUESTED** [review]

**Review by frivieremicroej — APPROVED** [review]
approve the min version

**Pull request merged**